dataland-frontend/src/components/general/AlignedActivitiesDataTable.vue

```diff
@@ -102,6 +102,24 @@ type MainColumnDefinition = {
   groupIndex: number;
 };
 
+const substantialContributionFields = [
```

Comment:
In general regarding this component: The formatting looks very strange when you have many nace codes. This should be fixed somewhere.
![image](https://github.com/d-fine/Dataland/assets/108792843/1c26b052-b924-44b7-8958-4652830a3478)

Reply:
The AlignedActivitiesDataTable.vue is being reworked in DALA-2307. Re-Check if issues persist after merge.

Reply:
@MarcinStanczyk7 will look at this

Reply:
fixed to 4 in row

Reply:
When trying to upload data, the activity fields seem to be gayed out.

Reply:
The whole upload form is build as if we are recording percent in 0 to 100 steps but display is made for 0 to 1

Reply:
This issue is now fully resolved (including that the dev environments again work as expected) such that the conversation should be closable